dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/impl/weld/injection/EEValidationUtils.java

```diff
@@ -0,0 +1,331 @@
+/*******************************************************************************
+ * Copyright (c) 2015 IBM Corporation and others.
```

Comment:
copyright needs updating ... derrived from 2015 code but definitely changed in 2018!